experiments/eccb2016/commands.sh

```diff
@@ -255,64 +181,23 @@ python compute_exclusivity.py -mf $UCEC_MUTATIONS -nc $NUM_CORES -k 3 \
 
-# FIGURE 1: Distribution of number of mutations per sample 
-python $SCRIPTS_DIR/sample_mutation_frequency_plot.py -mf $THCA_MUTATIONS \
-       $COADREAD_MUTATIONS $UCEC_MUTATIONS -c THCA COADREAD UCEC \
-       -o $FIGURES_DIR/Figure1.pdf
-
-# FIGURE 2: Mutation probability matrices
-python $SCRIPTS_DIR/weights_matrix.py -mf $THCA_MUTATIONS $COADREAD_MUTATIONS \
-       $UCEC_MUTATIONS -wf $THCA_WEIGHTS $COADREAD_WEIGHTS $UCEC_WEIGHTS \
-       -c THCA COADREAD UCEC -o $FIGURES_DIR/Figure2.pdf
-
-# FIGURE 3 and TABLE 1: Comparison of exact and saddlepoint for unweighted test
-python $SCRIPTS_DIR/unweighted_comparison.py -c THCA COADREAD UCEC \
-       -sf $THCA_UNWEIGHTED_SADDLEPOINT_TRIPLES \
-       $COADREAD_UNWEIGHTED_SADDLEPOINT_TRIPLES \
-       $UCEC_UNWEIGHTED_SADDLEPOINT_TRIPLES\
-       -ef $THCA_UNWEIGHTED_EXACT_TRIPLES $COADREAD_UNWEIGHTED_EXACT_TRIPLES \
-       $UCEC_UNWEIGHTED_EXACT_TRIPLES\
-       -ff $FIGURES_DIR/Figure3.png -tf $TABLES_DIR/Table1.tsv
-
-# FIGURE 4 and TABLE 3: Pairs
-python $SCRIPTS_DIR/pairs_summary.py -c THCA THCA THCA THCA COADREAD \
-       COADREAD COADREAD COADREAD UCEC UCEC UCEC UCEC -np $PAIRS_PERMUTATIONS \
-       -pf $THCA_PERMUTATIONAL_PAIRS $THCA_WEIGHTED_EXACT_PAIRS \
-       $THCA_WEIGHTED_SADDLEPOINT_PAIRS $THCA_UNWEIGHTED_EXACT_PAIRS \
-       $COADREAD_PERMUTATIONAL_PAIRS $COADREAD_WEIGHTED_EXACT_PAIRS \
-       $COADREAD_WEIGHTED_SADDLEPOINT_PAIRS $COADREAD_UNWEIGHTED_EXACT_PAIRS \
-       $UCEC_PERMUTATIONAL_PAIRS $UCEC_WEIGHTED_EXACT_PAIRS \
-       $UCEC_WEIGHTED_SADDLEPOINT_PAIRS $UCEC_UNWEIGHTED_EXACT_PAIRS \
-       -ff $FIGURES_DIR/Figure4.pdf -tf $TABLES_DIR/Table2.tsv $TABLES_DIR/Table3.tsv
-
-# FIGURE 5: Unweighted vs. Permutational (N=10^6)and Weighted (N=10^3) vs.
-# Permutational
-
-python $SCRIPTS_DIR/triple_pval_scatter.py -np $TOTAL_PERMUTATIONS \
-       -o $FIGURES_DIR/Figure5.png -uwf $COADREAD_UNWEIGHTED_EXACT_TRIPLES \
-       $THCA_UNWEIGHTED_EXACT_TRIPLES $UCEC_UNWEIGHTED_EXACT_TRIPLES \
-       -wf $COADREAD_WEIGHTED_SADDLEPOINT_TRIPLES \
-       $THCA_WEIGHTED_SADDLEPOINT_TRIPLES $UCEC_WEIGHTED_SADDLEPOINT_TRIPLES \
-       -pf $COADREAD_PERMUTATIONAL_TRIPLES $THCA_PERMUTATIONAL_TRIPLES \
-       $UCEC_PERMUTATIONAL_TRIPLES
-
 # TABLE 4: THCA results
 echo "THCA"
 python $SCRIPTS_DIR/results_table.py -lf $GENE_LENGTH_FILE \
        -mf $THCA_MUTATIONS -wf $THCA_WEIGHTED_SADDLEPOINT_TRIPLES \
-       -uf $THCA_UNWEIGHTED_EXACT_TRIPLES -nt 5 -lt $LENGTH_THRESHOLD \
+       -uf $THCA_UNWEIGHTED_SADDLEPOINT_TRIPLES -nt 5 -lt $LENGTH_THRESHOLD \
        -f $FDR_CUTOFF -o $TABLES_DIR/Table4
 
 # TABLE 5: COADREAD results
 echo "COADREAD"
 python $SCRIPTS_DIR/results_table.py -lf $GENE_LENGTH_FILE \
        -mf $COADREAD_MUTATIONS -wf $COADREAD_WEIGHTED_SADDLEPOINT_TRIPLES \
-       -uf $COADREAD_UNWEIGHTED_EXACT_TRIPLES -nt 5 -lt $LENGTH_THRESHOLD \
+       -uf $COADREAD_UNWEIGHTED_SADDLEPOINT_TRIPLES -nt 5 -lt $LENGTH_THRESHOLD \
```

Comment:
I'm not sure this is a good idea. It's one thing if `experiments/eccb2016/commands.sh` doesn't run unless you are at a certain tagged commit. It's another if it runs and produces something different, don't you think?